### PR TITLE
No jira add bpk bottom nav

### PR DESCRIPTION
**Pull request by bvitaliyg**
+ [ ] Check this if you have read and followed the [contributing guidelines](https://github.com/Skyscanner/backpack/blob/master/CONTRIBUTING.md)

Remember to include the following changes:
+ [ ] [Links platform metadata](https://github.com/Skyscanner/backpack-docs/blob/master/docs/src/layouts/links.js)

_If you are curious about how we review, please read through the [code review guidelines](https://github.com/Skyscanner/backpack/blob/master/CODE_REVIEW_GUIDELINES.md)_

### CODE REVIEW

File: docs/src/routes/Routes.js [hunk 1]
```diff
@@ -212,6 +213,10 @@ export const ROUTES_MAPPINGS = [
         path: ROUTES.BOTTOM_SHEET,
         component: BottomSheetPage,
       },
+      {
```
Comment (olliecurtis) [inline docs/src/routes/Routes.js]: Can this be moved to before the `BOTTOM_SHEET` for alphabetical order?

File: docs/src/routes/Routes.js [hunk 2]
```diff
@@ -60,6 +60,7 @@ import BadgePage from '../pages/BadgePage';
 import BannerAlertPage from '../pages/BannerAlertPage';
 import BarchartsPage from '../pages/BarchartsPage';
 import BottomSheetPage from '../pages/BottomSheetPage';
+import BottomNavPage from '../pages/BottomNavPage';
```
Comment (olliecurtis) [inline docs/src/routes/Routes.js]: Can this be moved to before the BOTTOM_SHEET for alphabetical order?

File: docs/src/layouts/links.js [hunk 3]
```diff
@@ -38,6 +38,12 @@ const ComponentsLinks = [
         children: 'Bottom sheet',
         tags: ['ios'],
       },
+      {
+        id: 'BOTTOM_NAV',
```
Comment (olliecurtis) [inline docs/src/layouts/links.js]: Can this be moved to before the BOTTOM_SHEET for alphabetical order?

File: docs/src/pages/AndroidBottomNavPage/AndroidBottomNavPage.js [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+    ],
+  },
+];
+
+const AndroidBottomNavPage = ({ ...rest }) => (
+  <DocsPageBuilder
+    title="Bottom Nav"
+    components={components}
+    readme={readme}
+    {...rest}
```
Comment (georgegillams) [inline docs/src/pages/AndroidBottomNavPage/AndroidBottomNavPage.js]: Can we add `androidDocId="net.skyscanner.backpack.bottomnav"`

File: docs/src/pages/AndroidBottomNavPage/AndroidBottomNavPage.js [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+      },
+    ],
+  },
+];
+
+const AndroidBottomNavPage = ({ ...rest }) => (
+  <DocsPageBuilder
+    title="Bottom Nav"
+    components={components}
+    readme={readme}
```
Comment (georgegillams) [inline docs/src/pages/AndroidBottomNavPage/AndroidBottomNavPage.js]: Can we add `androidDocId="net.skyscanner.backpack.bottomnav"`

File: docs/src/layouts/links.js (hunk 3 above)
Comment (georgegillams) [reply docs/src/layouts/links.js]: Note it automatically orders them on the site, so this is a minor thing

**Pull request merged**